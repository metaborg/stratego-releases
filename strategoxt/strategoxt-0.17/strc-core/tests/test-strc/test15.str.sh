#!/bin/sh

${STRC} -i test15.str -la stratego-lib || exit 1 

rm -f test15.str.actual
./test15 2>&1 | tail -n 2 | head -n 1 > test15.str.actual

diff test15.str.expected test15.str.actual || (echo "Result not expected; test failed"; exit 1)
