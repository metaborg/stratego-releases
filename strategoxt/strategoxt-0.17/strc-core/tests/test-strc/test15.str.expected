[ identity crisis | critical ] Internal error: with clause failed unexpectedly in rule 'main'
